src/client/components/ReviewPost/ListOfReviewPostComponent/ListOfReviewPostComponent.stories.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { ListOfReviewPostComponent } from './ListOfReviewPostComponent';
+import profileImage1 from '../../../assets/images/portraits/man1.jpg';
+import profileImage2 from '../../../assets/images/portraits/man2.jpg';
+
+
+export default {
+  title: 'ListOfReviewPostComponent',
+  component: ListOfReviewPost,
+};
+
+export const ListOfReviewPost = () => (
+  <ListOfReviewPostComponent
+    postedReviews={[
+      {
+        id:1,
+        profileImageUrl:`${profileImage1}`,
+        fullName: 'Hamed Karachi',
+        review:
+          'I reaaly like all parts of CV. It is clearly defined in which sections youi have abilities and skills. Just try to describe more about your background before when you start working as a project manager',
+        createdDate: new Date(),
+      },
+      {
+        id:2,
+        fullName: 'Mahmood Ehsani',
+        review:
+          'I believe that introduction section has an opportunity to be imporved in a matter that if recruiters wanted to know about your backgoround that can easily find out. try to remove some unrelevant thins from your CV as well.',
+        createdDate: new Date(),
+        },
+      {
+        id:3,
+        profileImageUrl:`${profileImage2}`,
+        fullName: 'Christoffer Tawn',
+        review:
+          'Everything is placed perfectly in CV except your picture. In my opinion it is not so attractive and energetic. try to change it with the smily one and a little bit formal as you want to introduce yourself as a project manager.',
+        createdDate: '2020/04/20, 23:00:00',
+        },
+      {
+        id:4,
```

Comment:
so `profileImageUrl` is optional to pass?

Reply:
@bytewiz I just wanted to show in story book how it looks like without profile image.